### DISCUSSION TITLE
Billing Issue

### DISCUSSION THREAD

**Discussion by joshuahufford** [community/community#52817] [Other Feature Feedback, Questions, & Ideas]
Hello.  I am writing here to see if there is a better way to contact support from a billing issue.  I was upgrading an organization to a team plan and the system charged my credit card $13,200 for an annual subscription with 300 seats.  I selected the annual subscription, but since we are just starting this organization, I selected 1 seat.

Is there anyone who can point me in a better direction than GitHub support or is there an employee who monitors these discussion who can assist?

Any help would be greatly appreciated!

**Comment by tuves** [comment]
Hi @joshuahufford! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by joshuahufford** [comment]
Thats why I'm here its been 7 days and haven't heard anything.  Is can I get my support ticket (#2099470) elevated?
